.travis.yml

```diff
@@ -33,7 +33,13 @@ script:
   - gpg2 --version
   - echo $PATH
   - which gpg2
-  - tox -e $TESTENV --recreate -- --no-test-cache --with-gpg2 tests/
+  # - tox -e $TESTENV --recreate -- --no-test-cache --with-gpg2 tests/
+  - cd core
```

Comment:
why another "cd core"?  There is already one above and the travis build fails because of this double "cd core".